examples/burial-form/PreviousNames.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import {
+  FullNameField,
+  Page,
+  RadioGroup
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+import { ExpandingGroupClass } from '../Constant';
+
+export default function PreviousNames(props) {
+  useEffect(() => {
```

Comment:
Can we remove this? I think it was causing stuttering in Firefox.

Reply:
you mean useEffect?